elfpy/utils/apeworx_integrations.py

```diff
@@ -317,8 +334,7 @@ def get_transfer_single_event(tx_receipt: ReceiptAPI) -> ContractLog:
 
 
 def get_pool_state(tx_receipt: ReceiptAPI, hyperdrive_contract: ContractInstance):
-    """
-    Return everything returned by `getPoolInfo()` in the smart contracts.
+    """Return everything returned by `getPoolInfo()` in the smart contracts.
```

Comment:
I _think_ you need the `r"""` syntax when using backticks to delineate code.

Reply:
oh right ya that sounds right. I think I was reading these as single quotes for a while.